pkgs/applications/finance/irpf/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib, stdenvNoCC, jdk11, makeWrapper, hicolor-icon-theme, fetchzip, fetchgit }:
+
+let
+  year = "2022";
+  pkgbuild = fetchgit rec {
+    url = "https://aur.archlinux.org/irpf.git/";
+    rev = "dc76a6366de72e236c91986a4c2a467d2472749f";
+    sha256 = "1hql5ljwq8g9h0kbgpfyss7mcmp58pzvjs5dydxcs39lxcnf972c";
+  };
+in
+stdenvNoCC.mkDerivation rec {
+  pname = "irpf";
+  version = "2.22.1.0";
+
+  src = fetchzip rec {
+    url = "https://downloadirpf.receita.fazenda.gov.br/irpf/${year}/irpf/arquivos/IRPF${year}-1.0.zip";
```

Comment:
```suggestion
  src = let
      year = lib.head (lib.splitVersion version);
  in fetchzip {
    url = "https://downloadirpf.receita.fazenda.gov.br/irpf/${year}/irpf/arquivos/IRPF${version}.zip";
```

Reply:
hm, I didn't even know this was possible. Thx!